src/containers/TopicPage/TopicContainer.tsx

```diff
@@ -155,6 +159,17 @@ export const TopicContainer = ({ node, subjectType }: TopicContainerProps) => {
               {node.relevanceId === RELEVANCE_SUPPLEMENTARY && (
                 <Badge colorTheme="neutral">{t("navigation.additionalTopic")}</Badge>
               )}
+              {!!node.path && !!articleId && (
+                <AddResourceToFolderModal
+                  resource={{
+                    id: articleId,
+                    path: node.path,
```

Comment:
Vi kan også begynne å lagre node.url, evt sjekke på enablePrettyUrls først.

Reply:
For en fremtidig migrering, er det ikke like greit å beholde alle som path inntil videre?

Reply:
Har ingenting å sei. Migrering med url vil berre gi 404 i tax og dermed ingen endring.